source/transfer_list.rst

```diff
@@ -447,6 +447,9 @@ The following entry types are currently defined:
 - single HOB block entry: tag_id = 2 (:numref:`hob_block_entry`).
 - HOB list entry: tag_id = 3 (:numref:`hob_list_entry`).
 - ACPI table aggregate entry: tag_id = 4 (:numref:`acpi_aggr_entry`).
+- OP-TEE pagable part entry: tag_id = 0x20 (:numref:`optee_pagable_part_entry`).
+- TOS FW config entry: tag_id = 0x21 (:numref:`tos_fw_config_entry`).
+- OPTEED AARCH entry: tag_id = 0x22 (:numref:`opteed_aarch_entry`).
```

Comment:
I don't think it's scalable to list every single TE here. Perhaps just just link to the OP-TEE section rather than list all OP-TEE TEs.

Reply:
Done.